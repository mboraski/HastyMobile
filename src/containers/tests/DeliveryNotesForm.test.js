import 'react-native';
import React from 'react';
import { shallow } from 'enzyme';
import configureStore from 'redux-mock-store';

import DeliveryNotesForm from '../DeliveryNotesForm';

describe('DeliveryNotesForm', () => {
    const middlewares = [];
    const mockStore = configureStore(middlewares);
    it('renders correctly', () => {
        const initialState = {
            checkout: { notes: 'notes' }
        };
        const props = {
            anyTouched: false,
            pending: false,
            submitting: false,
            asyncValidating: false,
            invalid: false,
            pristine: true,
            fields: {
                notes: {
                    value: 'notes',
                    touched: false,
                    error: null
                }
            }
        };
        const wrapper = shallow(<DeliveryNotesForm {...props} />, {
            context: { store: mockStore(initialState) }
        });
        const render = wrapper.dive();
        expect(render).toMatchSnapshot();
    });
});
