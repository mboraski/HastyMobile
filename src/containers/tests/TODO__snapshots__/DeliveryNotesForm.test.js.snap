// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeliveryNotesForm renders correctly 1`] = `
<Connect(Form(DeliveryNotesForm))
  anyTouched={false}
  asyncValidating={false}
  destroyOnUnmount={true}
  enableReinitialize={false}
  fields={
    Object {
      "notes": Object {
        "error": null,
        "touched": false,
        "value": "notes",
      },
    }
  }
  forceUnregisterOnUnmount={false}
  form="DeliveryNotes"
  getFormState={[Function]}
  initialValues={
    Object {
      "notes": "notes",
    }
  }
  invalid={false}
  keepDirtyOnReinitialize={false}
  onSubmit={[Function]}
  pending={false}
  persistentSubmitErrors={false}
  pristine={true}
  pure={true}
  setDeliveryNotes={[Function]}
  shouldAsyncValidate={[Function]}
  shouldValidate={[Function]}
  submitting={false}
  touchOnBlur={true}
  touchOnChange={false}
/>
`;
